tests/python/relay/test_op_level5.py

```diff
@@ -67,6 +69,7 @@ def verify_resize(dshape, scale, method, layout):
     for method in ["bilinear", "nearest_neighbor"]:
         for layout in ["NHWC", "NCHW"]:
             verify_resize((1, 4, 4, 4), 2, method, layout)
+            verify_resize((2, 8, 17, 20), 2, method, layout)
```

Comment:
Can we also add a test case with a different scale?

Reply:
Just added another scale